src/NexusMods.App.UI/Pages/MyLoadouts/MyLoadoutsView.axaml

```diff
@@ -0,0 +1,70 @@
+<reactiveUi:ReactiveUserControl x:TypeArguments="myLoadouts:IMyLoadoutsViewModel" xmlns="https://github.com/avaloniaui"
+                                xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                                xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+                                xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+                                xmlns:reactiveUi="http://reactiveui.net"
+                                xmlns:myLoadouts="clr-namespace:NexusMods.App.UI.Pages.MyLoadouts"
+                                xmlns:panels="clr-namespace:Avalonia.Labs.Panels;assembly=Avalonia.Labs.Panels"
+                                xmlns:gameLoadoutsSectionEntry="clr-namespace:NexusMods.App.UI.Pages.MyLoadouts.GameLoadoutsSectionEntry"
+                                xmlns:ui="clr-namespace:NexusMods.App.UI"
+                                mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+                                x:Class="NexusMods.App.UI.Pages.MyLoadouts.MyLoadoutsView">
+    <Design.DataContext>
+        <myLoadouts:MyLoadoutsDesignViewModel />
+    </Design.DataContext>
+
+
+    <ScrollViewer>
+        <panels:FlexPanel x:Name="MyLoadoutsOuterFlexPanel">
+
+            <TextBlock x:Name="PageDescriptionTextBlock"
+                       Text="Create game Loadouts which contain their own individual mod preferences and settings. Switch between Loadouts and apply the one you want to play." />
```

Comment:
What's with this text?

Reply:
I'll do a translations pass at the end if that is the question. All the texts are from figma

Reply:
What is this text even supposed to mean? @captainsandypants is this placeholder text?

Reply:
> Create game Loadouts which contain their own individual mod preferences and settings. Switch between Loadouts and apply the one you want to play.

My issue with this text, is that it's partially wrong and doesn't explain what a Loadout is, if that was the goal here. It's great to have tutorials or help text for beginners, but this just feels so empty. I can't really put it in words, this is like when you have an assignment with a minimum word requirement, and you need padding to fill it out.

Reply:
Which part is wrong? This is both generic in that it should cover all games but also succinct in explaining what the user can do with Loadouts.

Reply:
"Create new game Loadouts, each containing a separate set of mods. Switch between them by Applying the one you wish to use."

Reply:
> Which part is wrong? This is both generic in that it should cover all games but also succinct in explaining what the user can do with Loadouts.

Loadouts don't "contain their own individual mod preferences and settings". What is even the difference between "mod preferences" and "settings" here? For the laymen, a Loadout contains their installed mods. The text doesn't even mention that, it just talks about preferences.

"Switch between Loadouts and apply the one you want to play." this is also problematic because it doesn't really fit the context of the page. From how I understand the page, you can create and delete loadouts here. Switching between Loadouts is done via the Spine, and applying and playing Loadouts is done via the buttons in the left menu.

I feel like there are much better alternatives for text that quickly explains what a Loadout is and what you can do on this page like

> Here you can create and remove Loadouts. You can install Mods into a Loadout using the Library.

That's about as short and easy to understand as it gets.

Reply:
We agreed not to use the word "Install" for adding mods to a mod list (although I would be happy to change that as I think it's the closest thing to representing what you're doing). But either way, you can also "install" mods to collections, and you could have more than one collection in a loadout, and you can switch between those collections. The key differentiator of a loadout is that it contains more than just different mods.

For the second part I could see how that could confuse users into thinking they can apply a loadout here.

How about:

> Create personalised game Loadouts with individual mod setups. You can install Mods into a Loadout using the Library.

Reply:
I would probably skip the library part. I though that mentioning that Applying is how you can switch between loadouts was one of the main points of the original message. 
Otherwise users will see cards with the Tick and might wonder how to set on another loadout.

Reply:
> Create personalised game Loadouts with individual mod setups.

Shouldn't use both "personalised" and "individual" in the same sentence, kinda the same thing. "personalised" means "adapted to the needs of an individual". Both of these adjectives can be removed, I don't think they add any value here. Without them, we'd end up with "Create game Loadouts with mod setups". I don't think "game Loadouts" has to be prefixed like this, the UI groups Loadouts by game so it should be clear that Loadouts are game specific. "mod setups" is very ambiguous and doesn't really convey what a Loadout is. Maybe we don't need a sentence saying you can "create Loadouts" on this page. The UI should make it very clear that you can do that, since we have a card that literally says "Create Loadout".

> You can install Mods into a Loadout using the Library.

I forgot about "add", "You can add Mods from the Library to the Loadout" maybe?

Reply:
> Create game Loadouts which contain their own individual mod preferences and settings. Switch between Loadouts and apply the one you want to play.

From reading the original message, my guess on the purpose of it would be:
- Describe what a Loadout is to new users that don't know what it should mean in the App context.
- Give a hint as to how switching between loadouts works (through Apply).
- Contextualize the info about loadouts within the purpose of the page (creating new loadouts).

I think the original text doesn't intuitively or accurately describe what a loadout is, which would be an independent set of installed mods or collections.

The important thing to communicate here is the fact that mods inside different Loadouts are separate and independent from each other. 
This is important because in Vortex (and MO2) Profiles exist as a similar concept, but they share the mods and instead only differ per mod and plugin order as well as various configurations and enabled states.

Reply:
How about:

> Loadouts enable you to switch between various play styles by customising each with different mods and preferences. Simply select and Apply the Loadout you wish to use.

This explains what their purpose is and how to use them.

Reply:
I would skip the preferences. It has no context specific meaning in the app (we have no named "Preferences" concept), or outside the app, as there are too many things that preferences could apply to.

Reply:
The game specific bit then explains in more detail what preferences are supported. Preferences is intentionally generic and just means the things you can change to your preference, i.e. the preferences I've chosen in a config file or FOMOD. It doesn't have to be a concept, the meaning of the word is understood.

We could go with "options" instead of "preferences" if we think we will want to use preferences for something more specific later.

Reply:
I would suggest "configurations" instead
> Loadouts enable you to switch between various play styles by customising each with different mods and configurations. Simply select and Apply the Loadout you wish to use.

Reply:
I'm happy with that. Let's go with it.

Reply:
Done. 
I personally am not partial to the "play styles" part as that wouldn't be the the main use case for the majority of people using the feature.  But coming up with names and wording is hard.

Reply:
We can update the game specific text based on this too. 

Supports individual: Mods and mod configurations.
Shares: Library and saved games.